en/contents.rst

```diff
@@ -43,7 +43,7 @@ Contents
     core-libraries/internationalization-and-localization
     core-libraries/logging
     core-libraries/form
-    controllers/components/pagination
+    controllers/pagination
```

Comment:
You might get warnings that this document isn't in a TOC. If so you can add a hidden toc

```rst

.. toctree::
    :hidden:
    
    controllers/components/pagination
```

Should fix the warnings.

Reply:
The warnings were due to duplicate labels, fixed now.